src/libraries/Microsoft.PowerFx.Core/Types/DType.cs

```diff
@@ -1881,7 +1881,6 @@ public virtual bool Accepts(DType type, out KeyValuePair<string, DType> schemaDi
                         type.Kind == DKind.Media ||
                         type.Kind == DKind.Blob ||
                         type.Kind == DKind.Unknown ||
-                        type.Kind == DKind.Guid ||
```

Comment:
> type.Kind == DKind.Guid [](http://example.com/codeflow?start=23&length=24)

Will this cause any issues on PAClient?

Reply:
Yes, possibly. I'm double checking if there's any impact there. Should be safe, as long as we implement the coercion operator in JS as well.